src/main/java/commands/chat/Vote3.java

```diff
@@ -190,6 +190,7 @@ public static void handleReaction(MessageReactionAddEvent event) {
                 if (reactions.contains(event.getReaction().getEmote().getName())) {
                     System.out.println("TEST 4");
                     addVote(guild, event.getMember(), reactions.indexOf(event.getReaction().getEmote().getName()) + 1);
+                    event.getReaction().removeReaction(event.getUser()).queue();
```

Comment:
Das geht *(mittlerweile?)*? 
Hatte das früher in meinen Tests so drin und da hat er **alle** Reactions, also den ganzen Stack + die ursprünglche vom Bot gelöscht. 
Ist das jetzt anders?

Reply:
removeReaction(event.getUser()) da hier der User übergeben wird wird nur die Reaction vom User gelöscht